src/main/scala/tech/cryptonomic/conseil/io/MainOutputs.scala

```diff
@@ -165,8 +165,8 @@ object MainOutputs {
 
   /* custom display of each configuration type */
   private val showPlatformConfiguration: PartialFunction[PlatformConfiguration, String] = {
-    case TezosConfiguration(_, TezosNodeConfiguration(host, port, protocol, prefix)) =>
-      s"node $protocol://$host:$port/$prefix"
+    case TezosConfiguration(_, TezosNodeConfiguration(host, port, protocol, prefix, chainEnv)) =>
+      s"node $protocol://$host:$port/$prefix/$chainEnv"
```

Comment:
shouldn't we put the `/chains/` path fragment here, too, for consistency?

Reply:
It's only for debugging purpose so I would leave it as short as possible.